plugins/multi-projects-extension.js

```diff
@@ -506,12 +510,10 @@ window.plugin.mpe.setupCSS = function(){
 
 
 var setup = function(){
-  window.pluginCreateHook('mpe');
```

Comment:
Is the `mpe` hook no longer needed?

Reply:
Hook is needed, but there is no need to pre-register it.